### PR TITLE
solution

### PR DESCRIPTION
**Pull request by tanyakovchuk**
[DEMO LINK](https://taya294.github.io/react_movies-list-filter/)

### CODE REVIEW

**Review by Grygoriy-Shytikov — APPROVED** [review]
Виглядає добре! 👍

File: src/App.js [hunk 1]
```diff
@@ -4,16 +4,35 @@ import { MoviesList } from './components/MoviesList';
...
+
+  searchFilm = (event) => {
+    this.setState({
+      userRequest: event.target.value,
+    });
+  }
 
   render() {
+    const visibleFilms = moviesFromServer.filter(
+      m => m.title.toLowerCase().includes(
```
Comment (varseniuk) [inline src/App.js:19]:
```suggestion
      movie => movie.title.toLowerCase().includes(
```
Better to use meaningful namings

File: src/App.js [hunk 2]
```diff
@@ -4,16 +4,35 @@ import { MoviesList } from './components/MoviesList';
...
+
     return (
       <div className="page">
         <div className="page-content">
           <div className="box">
             <div className="field">
               <label htmlFor="search-query" className="label">
                 Search movie
+                {' '}
+                {this.state.filmName}
```
Comment (varseniuk) [inline src/App.js:35]: 👍

Comment (varseniuk) [inline src/App.js:35]:
```suggestion
                {this.state. userRequest}
```
Looks like you don't have `filmName` in the state. That value called `userRequest`

**Review by varseniuk — APPROVED** [review]
Great job!

**Pull request closed without merging**